bin/vaultclient

```diff
@@ -213,20 +215,28 @@ program
 
 program
     .command('update-account-quota')
-    .option('--account-name <NAME>', 'Name of the account')
+    .option('--account-name [NAME]', 'Name of the account')
```

Comment:
nit: keeping `<NAME>` was better IMO, the brackets are more often used for alternative or lists...
In any case, should be consistent with next line, where `<QUOTA>` is used (and be consistent in every command docs...)

Reply:
done here: https://github.com/scality/vaultclient/pull/442/files#diff-6ed3f39879bec9944e1a6e9d0d3c1bcbc7209a7eab826e5c3819048e4841163dR218